src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -330,7 +346,8 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
             {
                 id:"survival",
                 hide:()=>{
-                    return this.resultsViewPageStore.studies.result!.length > 1;
+                    return !this.resultsViewPageStore.survivalClinicalData.isComplete ||
+                        this.resultsViewPageStore.survivalClinicalData.result.length === 0;
```

Comment:
Won't this load the whole survival data? I thought we were going to call the `META` projection and check the `total-count` header so that the data itself is still lazy loaded when the tab is clicked.